src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+
+"*I would absolutely play vox in SS14 lol*"
+
+"*i think i have been voxpilled*"
+
+[A vox discussion that I did not start I swear ](https://discord.com/channels/310555209753690112/310555209753690112/1200361375856332830)
+
+[A vox discussion](https://discord.com/channels/310555209753690112/310555209753690112/1200411180779438081)
+
+[Discord thread where we are attempting to contain the vox talk](https://discord.com/channels/310555209753690112/1200498334587179018)
+
+
+## Design Pillars
+
+Disclaimer, all my info is secondhand. If you have any input, feel free to nitpick about the details.
+
+My best attempt to boil down (or am I re-interpreting?) what the vox are about, are these three pillars: Weird, Difficult, and Cooperative.
+Everything designed about/for them should aim to channel or at the very least not contradict these three ideas.
+
+
+#### Weird
+They are a weird species, which is perhaps a strange thing to specify in a scifi setting with moth people and space carps, but even so. They are probably cybernetic and their biology is particularly alien I mean who dies from oxygen?! They scream, dammit why do they scream?? Why are they drinking the welder fluid??? While care should be taken that their features are not (encouraging to be) annoying, it's fine if they are a little Weird.
+
+#### Difficult
+They are not intended to be perfectly balanced, or even necessarily what you might call "on par". Their main "feature" is in itself very punishing. If they overall fall towards the weaker side of the meta, that's just the cost of being vox. While they can and should have some positives, they should not be given big buffs just to "balance out" their shortcomings. They should not be prohibitively difficult, but if they are a less popular choice, that fits just fine into Difficult.
+
+#### Cooperative
+Historically, due to the difficulty of surviving a round as a vox, they had a tendency to rely on each other to get by in a world that was just not built for them. When no one else "got it", vox players turned for support to those who did. (Or so I'm told). It is the position of this design that this cooperative aspect of them is something to be preserved, and it being species specific is not something that should either be encouraged or discouraged mechanically, leaving it to the players to decide.
+Obviously, "just designating" how a species is supposed to play like is an impossible, lofty goal, but nevertheless their features could be designed in a way to help with that
+This was the hardest pillar to phrase and will be the hardest to come up with ideas for. But if a feature works better with other players, does something for other players rather than you, or is a flaw that is more easily counteracted with external help rather than just having the right tool in your pocket, then that feature is Cooperative.
+
+
+## Specific Features and Ideas 
```

Comment:
Want to add a suggestion: make them *very very slightly* faster than normal, maybe 5-6%.

This would fit perfectly with the little dudes, fragile and tiny but fast.